updating/understanding-openshift-updates.adoc

```diff
@@ -13,3 +13,12 @@ The graph is based on recommended, tested update paths from a specific version.
 After the CVO receives the update image from the registry, the CVO then applies the changes.
 
 include::modules/update-common-terms.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[discrete]
+[id="additional-resources_understanding-updates"]
+== Additional resources
+
+* xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[About the OpenShift Update Service]
+* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Understanding update channels and releases]
```

Comment:
Should this title be just "Update channels and releases" to match the section's heading?